manim/mobject/three_dimensions.py

```diff
@@ -32,6 +32,24 @@ def __init__(self, shade_in_3d=True, **kwargs):
 
 
 class ParametricSurface(VGroup):
+    """Creates a Parametric Surface
+
+    Exanples
+    --------
+    .. manim:: ParaSurface
+        :save_last_frame:
+
+        class ParaSurface(ThreeDScene):
+            def func(self, u, v):
+                return np.array([np.cos(u) * np.cos(v), np.cos(u) * np.sin(v), u])
+
+            def construct(self):
+                axes = ThreeDAxes()
```

Comment:
I feel like the current x-axis range - (-6, 6) is way too big. Not really an issue, but it looks good to me when the range and length are adjusted to - `ThreeDAxes(x_range=[-4,4], x_length=8)`. 

The final render together with the suggestion below looks like this: 
![ParaSurface_ManimCE_v0 7 0](https://user-images.githubusercontent.com/73361366/121772595-e22a8300-cb93-11eb-84fc-e72d7a4dda83.png)

Reply:
I took the suggestions into consideration. Thank you.